src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -518,5 +525,35 @@ public static String getInstanceAttributeNameForOutputReport(GKInstance instance
         }
         return attributeInstance != null ? attributeInstance.getDisplayName() : "N/A";
     }
-    
+
+    /**
+     * @return - long, DBID for 'SARS-CoV-2 Infection' Pathway instance.
+     */
+    public static long getCoV2InfectionPathwayDbId() {
+        return cov2InfectionPathwayDbId;
+    }
+    /**
+     * @return - long, DBID for 'Human SARS coronavirus' (CoV-1) Species instance.
+     */
+    public static long getCoV1SpeciesDbId() {
+        return cov1SpeciesDbId;
+    }
+    /**
+     * @return - long, DBID for 'Severe acute respiratory syndrome coronavirus 2' (CoV-2) Species instance.
+     */
+    public static long getCoV2SpeciesDbId() {
+        return cov2SpeciesDbId;
+    }
+    /**
+     * @return - long, DBID for 'severe acute respiratory syndrome' Disease instance.
+     */
+    public static long getCoV1DiseaseDbId() {
+        return cov1DiseaseDbId;
+    }
+    /**
+     * @return - long, DBID for 'COVID-19' Disease instance.
+     */
+    public static long getCoV2DiseaseDbId() {
+        return cov2DiseaseDbId;
+    }
```

Comment:
Do you think the constants themselves can be made public in place of getter methods?

Reply:
Done